### PR TITLE
added json files for initial  and mnc job questions

### PR DESCRIPTION
**Pull request by sanjay212003**

### CODE REVIEW

File: data/initial_questions.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "initial_questions",
+  "data": [
+    {
+      "index": 0,
+      "image": "  _____     \n /     \\    \n|  O  O |   \n|   |   |   \n|   _   |   \n \\_____/    \n     |      \n     |      \n  ____|____ \n |         | \n |         | \n  ___________",
+      "question": "The alarm rings at 6 AM. What’s your reaction?",
+      "options": [
+        { "text": "Snooze it for “just 5 more minutes.”", "points": { "money": 0, "health": -5, "happiness": 0 } },
```
Comment (TheShiveshNetwork) [inline data/initial_questions.json]: Check the file, https://github.com/TheShiveshNetwork/kernel-panic/blob/main/data/studyabroad.json
The points field are health, happiness and wealth. Not money!

File: data/initial_questions.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        { "text": "Jump out of bed, ready to conquer the world.", "points": { "money": 0, "health": 5, "happiness": 5 } },
+        { "text": "Turn it off, roll over, and go back to sleep.", "points": { "money": 0, "health": -5, "happiness": 10 } }
+      ]
+    },
+    {
+      "index": 1,
+      "image": "    _____        \n   /     \\      \n  |  O O  |     \n  |   |   |     \n  |  ___  |     \n   \\_____/      \n     |         \n     |         \n   ___|___     \n  |       |    \n  |       |    \n  _____________",
+      "question": "You’ve just started your first semester in engineering. How do you handle this new chapter?",
+      "options": [
+        { "text": "Devote your evenings to perfecting your notes and acing every assignment, even skipping the occasional hangout.", "points": { "money": 5, "health": -5, "happiness": 5 } },
```
Comment (TheShiveshNetwork) [inline data/initial_questions.json]: Fix this in every place.

File: data/mnc_job.json [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+          { "text": "Suggest someone more experienced for the role.", "points": { "money": 0, "health": 10, "happiness": 10 } },
+          { "text": "Offer to co-lead with a colleague to share the load.", "points": { "money": 10, "health": -5, "happiness": 5 } }
+        ]
+      },
+            {
+                "index": 12,
+                "image": "    ____       ____       ____\n   /    \\     /    \\     /    \\\n  |      |   |      |   |      |\n  |______|   |______|   |______|\n   |    |     |    |     |    |\n   |    |     |    |     |    |\n  __|____|____|____|____|____|__\n | | |  | | |  | | |  | |  | |",
+                "question": "Your team is underperforming. How do you address it?",
+                "options": [
+                    { "text": "Set up one-on-one meetings to understand issues.", "points": { "health": 0, "happiness": 15, "wealth": 0 } },
```
Comment (TheShiveshNetwork) [inline data/mnc_job.json]: Here it is correct. Cross check everywhere. Not adding any more comments.

**Review by TheShiveshNetwork — CHANGES_REQUESTED** [review]

File: data/mnc_job.json (hunk 3 above)
Comment (Abhishekvk04) [reply data/mnc_job.json]: corrected it.

**Comment by Abhishekvk04** [comment]
Rectified all error, Check once if there is still error or changes required.

**Review by TheShiveshNetwork — APPROVED** [review]

**Pull request merged**